packages/core/src/rules/prefer_is_not.ts

```diff
@@ -50,13 +53,43 @@ IF NOT variable = 42.`,
           continue;
         }
 
-        const message = "Prefer IS NOT to NOT IS";
-        issues.push(Issue.atToken(file, c.getFirstToken(), message, this.getMetadata().key, this.conf.severity));
+        const message = "Prefer expresion NOT to NOT expresion";
```

Comment:
I think something is missing in the message (probably the "IS") - also it's "expression" (two 's')

Reply:
I don't like the message with IS, because the IS is not there most of the time.
The condition could look like IF NOT 5 = 2., IF NOT 5 IN range or IF NOT number BETWEEN 3 AND 5.
But I agree that expression (even without the grammatical mistake) is not clear as well.

Reply:
I agree its not easy to understand, 

but keep it to the title in the style guilde, https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#prefer-is-not-to-not-is